Specs/Scene/GltfJsonLoaderSpec.js

```diff
@@ -0,0 +1,674 @@
+        bufferView: 0,
+        byteOffset: 0,
+        componentType: 5126,
+        type: "VEC3",
+        count: 1,
+        name: "accessor",
+        min: [0, 0, 0],
+        max: [0, 0, 0],
+        normalized: false,
+      },
+    ],
+    meshes: [
+      {
+        primitives: [
+          {
+            attributes: {
+              POSITION: 0,
+            },
+            mode: 4,
+            material: 0,
+          },
+        ],
+        name: "mesh",
+      },
+    ],
+    nodes: [
+      {
+        name: "node",
+        mesh: 0,
+        matrix: [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1],
+      },
+    ],
+    scene: 0,
+    scenes: [
+      {
+        nodes: [0],
+        name: "scene",
+      },
+    ],
+    materials: [
+      {
+        name: "default",
+        emissiveFactor: [0, 0, 0],
+        alphaMode: "OPAQUE",
+        doubleSided: false,
+      },
+    ],
+  };
+
+  var gltf2 = {
+    asset: {
+      version: "2.0",
+    },
+    buffers: [
+      {
+        uri: "external.bin",
+        byteLength: 12,
+      },
+    ],
+    bufferViews: [
+      {
+        buffer: 0,
+        byteOffset: 0,
+        byteLength: 12,
+      },
+    ],
+    accessors: [
+      {
+        bufferView: 0,
+        byteOffset: 0,
+        componentType: 5126,
+        type: "VEC3",
+        count: 1,
+        min: [0, 0, 0],
+        max: [0, 0, 0],
+      },
+    ],
+    meshes: [
+      {
+        primitives: [
+          {
+            attributes: {
+              POSITION: 0,
+            },
+          },
+        ],
+      },
+    ],
+    nodes: [
+      {
+        mesh: 0,
+        matrix: [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1],
+      },
+    ],
+    scene: 0,
+    scenes: [
+      {
+        nodes: [0],
+      },
+    ],
+  };
+
+  var gltf2Updated = {
+    asset: {
+      version: "2.0",
+    },
+    buffers: [
+      {
+        uri: "external.bin",
+        byteLength: 12,
+      },
+    ],
+    bufferViews: [
+      {
+        buffer: 0,
+        byteOffset: 0,
+        byteLength: 12,
+        byteStride: 12,
+        target: 34962,
+      },
+    ],
+    accessors: [
+      {
+        bufferView: 0,
+        byteOffset: 0,
+        componentType: 5126,
+        type: "VEC3",
+        count: 1,
+        min: [0, 0, 0],
+        max: [0, 0, 0],
+        normalized: false,
+      },
+    ],
+    meshes: [
+      {
+        primitives: [
+          {
+            attributes: {
+              POSITION: 0,
+            },
+            mode: 4,
+            material: 0,
+          },
+        ],
+      },
+    ],
+    nodes: [
+      {
+        mesh: 0,
+        matrix: [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1],
+      },
+    ],
+    scene: 0,
+    scenes: [
+      {
+        nodes: [0],
+      },
+    ],
+    materials: [
+      {
+        name: "default",
+        emissiveFactor: [0, 0, 0],
+        alphaMode: "OPAQUE",
+        doubleSided: false,
+      },
+    ],
+  };
+
+  function createGlb1(json) {
```

Comment:
Do we have a XxxxTester for creating GLBs like this? or does this not generalize enough for that to make sense?

Reply:
I don't think we do. Maybe once we get further in the Model.js refactor we'll see if this needs to be made a common function.